runtime/src/stakes.rs

```diff
@@ -533,6 +533,15 @@ pub(crate) mod tests {
         vote_pubkey: &Pubkey,
     ) -> (Pubkey, AccountSharedData) {
         let stake_pubkey = solana_sdk::pubkey::new_rand();
+        create_stake_account_with_id(stake, vote_pubkey, stake_pubkey)
+    }
+
+    //   add stake to a vote_pubkey                               (   stake    )
+    pub(crate) fn create_stake_account_with_id(
+        stake: u64,
+        vote_pubkey: &Pubkey,
+        stake_pubkey: Pubkey,
+    ) -> (Pubkey, AccountSharedData) {
```

Comment:
Maybe instead of adding a new method, just update `create_stake_account` so that the caller would always pass `stake_pubkey` as an argument. There are only 5 call sites that need to be updated.

Also then it wouldn't be necessary to return the stake pubkey again.

Like `) -> (Pubkey, ...) {` in `create_stake_account_with_id` is the same as `stake_pubkey` already passed in.

Reply:
done